src/test/java/com/github/mmolimar/kafka/connect/fs/task/FsSourceTaskTest.java

```diff
@@ -75,19 +80,31 @@ public void initTask() {
             OffsetStorageReader offsetStorageReader = PowerMock.createMock(OffsetStorageReader.class);
 
             EasyMock.expect(taskContext.offsetStorageReader())
-                    .andReturn(offsetStorageReader);
-
-            EasyMock.expect(taskContext.offsetStorageReader())
-                    .andReturn(offsetStorageReader);
-
-            EasyMock.expect(offsetStorageReader.offset(EasyMock.anyObject()))
-                    .andReturn(new HashMap<String, Object>() {{
-                        put("offset", (long) (NUM_RECORDS / 2));
-                    }});
-            EasyMock.expect(offsetStorageReader.offset(EasyMock.anyObject()))
-                    .andReturn(new HashMap<String, Object>() {{
-                        put("offset", (long) (NUM_RECORDS / 2));
-                    }});
+                    .andReturn(offsetStorageReader)
+                    .times(2);
+
+            // Every time the `offsetStorageReader.offset(params)` method is called we want to capture the offset params
```

Comment:
There may be a better way to use EasyMock to do this, but I could not find a way to do it in the EasyMock docs.